src/Coypu.AcceptanceTests/ApiExamples.cs

```diff
@@ -646,6 +646,7 @@ private static RemoteWebDriver CustomProfile()
 
         [TestCase("Windows 7", "firefox", "25")]
         [TestCase("Windows XP", "internet explorer", "6")]
+        [TestCase("Windows 10", "MicrosoftEdge", "14")]
         public void CustomBrowserSession(string platform, string browserName, string version)

```

Comment:
HI thanks for this! Did you actually run this test? I would have thought you'd get `BrowserNotSupported` unless you also added some code to construct the correct selenium driver here:

https://github.com/featurist/coypu/blob/c86ac64d3f82aeb184eeeca1ee10b6b9850e86f1/src/Coypu/Drivers/Selenium/DriverFactory.cs